### PR TITLE
 assignee min and max added

### LINKED ISSUE #29: Validate min and max of Asignee in PR
## Context
A team may decide that every PR should have at least an assignee. By default zero number of assignees is acceptable.

## Details
Support Advanced settings:
```
  assignee:
    min: 1
    max: 1
    message: 'Custom message...'
```

and simple:

```
assignee: 1 # min number of assignees.
```

### PR DESCRIPTION
**Pull request by shine2lay**
closes #29

### CODE REVIEW

File: lib/assignee.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const processor = require('../lib/processor')
+
+/**
+ * Determines if the the PR is mergeable based number of assignees
```
Comment (jusx) [inline lib/assignee.js]: Small grammar `based on number of assignees`

File: __tests__/assignee.test.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  validation = await assignee(createMockPR(2), null, config({config: configuration}))
+  expect(validation.mergeable).toBe(true)
+})
+
+test('checks that advance_setting max is working', async () => {
+  let configuration = `
+  mergeable:
+    assignee:
+      min: 2
```
Comment (jusx) [inline __tests__/assignee.test.js]: should be max: 2

File: __tests__/assignee.test.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  `
+
+  let validation = await assignee(createMockPR(1), null, config({config: configuration}))
+  expect(validation.mergeable).toBe(false)
+  expect(validation.description[0]).toBe('Assignee count is less than "2"')
```
Comment (jusx) [inline __tests__/assignee.test.js]: should be using max description.

File: __tests__/assignee.test.js (hunk 2 above)
Comment (jusx) [reply __tests__/assignee.test.js]: I think you have this test twice and is a copy paste mistake.

File: __tests__/assignee.test.js (hunk 3 above)
Comment (shine2lay) [reply __tests__/assignee.test.js]: removed the whole test case as it is a duplicate

**Pull request merged**